rs/nns/governance/src/network_economics.rs

```diff
@@ -4,9 +4,9 @@ use crate::pb::v1::{
 };
 use ic_nervous_system_common::{E8, ONE_DAY_SECONDS, ONE_MONTH_SECONDS};
 use ic_nervous_system_linear_map::LinearMap;
-use ic_nervous_system_proto::pb::v1::Percentage;
+use ic_nervous_system_proto::pb::v1::{Decimal as ProtoDecimal, Percentage};
 use icp_ledger::DEFAULT_TRANSFER_FEE;
-use rust_decimal::Decimal;
+use rust_decimal::Decimal as NativeDecimal;
```

Comment:
Seems like new terminology, in other places we used `TypePb` (sometimes `TypeProto`) and `Type`

and in some places `TypeInner` and `Type`

Shall we recycle one of the existing conventions rather than introducing an N+1st one?